src/org/labkey/test/tests/ProjectCreatorUserTest.java

```diff
@@ -164,9 +165,9 @@ public void testCreateProjectByTemplate() throws IOException, CommandException
         stopImpersonating();
 
         goToProjectHome(PROJECT_NAME_PC);
-        assertTrue(projectMenu().projectLinkExists(PROJECT_NAME_PC));
```

Comment:
Asserting project is present got deleted accidentally?

Reply:
This was intentional. `goToProjectHome` does essentially the same validation. Opening the project menu to check for the link is what was causing the intermittent failures.

Reply:
ok